.github/workflows/ci.yml

```diff
@@ -71,13 +71,13 @@ jobs:
       - name: Compile and Download
         run: yarn npm-run-all --max_old_space_size=4095 -lp compile "electron x64" # {{SQL CARBON EDIT}} Remove unused options playwright-install download-builtin-extensions
 
-      - name: Run Unit Tests (Electron)
+      - name: Run Core Unit Tests
```

Comment:
Add `{{SQL CARBON EDIT}} Rename to core for better clarity` to lines changed in this file

Reply:
Updated.